package.json

```diff
@@ -66,6 +67,7 @@
     "async-mutex": "^0.4.0",
     "axios": "^1.6.5",
     "bignumber.js": "^9.1.2",
+    "jest-extended": "^4.0.2",
```

Comment:
can we bump @openzeppeling/contracts to 5.0.2. Intellij says 5.0.1 is vulnerable.